src/main/haskell/language-kore/src/Data/Kore/ASTVerifier/PatternVerifier.hs

```diff
@@ -413,6 +419,23 @@ verifyVariableUsage variable _ verifyHelpers _ _ = do
         "The declared sort is different."
     return (variableSort variable)
 
+-- TODO: properly verify child pattern if it's not a stringLiteral
+verifyDomainValue
+    :: (MetaOrObject level)
+    => DomainValue Object
+    -> VerifyHelpers level
+    -> Set.Set UnifiedSortVariable
+    -> Either (Error VerifyError) (Sort Object)
+verifyDomainValue dv verifyHelpers declaredSortVariables =
+    case isMetaOrObject verifyHelpers of
```

Comment:
This function has some holes in testing, could you test what is worth testing (maybe everything)?